### PR TITLE
Added GrowRow()

### PR DESCRIPTION
**Pull request by carlosproiete**
Hello! I added a new method called GrowRow(int), to grow the selection not only in both ways, but downwards.

### CODE REVIEW

**Comment by igitur** [comment]
Please add some unit tests. Look for the unit tests for `Grow()` for an example.

File: ClosedXML/Excel/Ranges/IXLRangeBase.cs [hunk 1]
```diff
@@ -320,6 +320,13 @@ public interface IXLRangeBase : IXLAddressable
         /// <returns></returns>
         IXLRangeBase Grow(Int32 growCount);
 
+        /// <summary>
+        /// Grows this the current range by the specified number of rows.
+        /// </summary>
+        /// <param name="growRowCount">The grow count.</param>
+        /// <returns></returns>
+        IXLRangeBase GrowRow(Int32 growRowCount);
```
Comment (igitur) [inline ClosedXML/Excel/Ranges/IXLRangeBase.cs:328]: I think `GrowDown` is a better name. Agree?

**Review by igitur — CHANGES_REQUESTED** [review]
Please also add unit tests.

**Comment by jahav** [comment]
Closing without merge.

I don't think it's worth expanding API surface for this. The proposed method could easily be implemented in client code as an extension method. Also, there are no unit tests and semantic is unclear, doc says that it expands the current range, but method returns a new one without expanding the original one.

**Pull request closed without merging**